Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -508,13 +508,15 @@ def update_trusted_images(self, data: dict):
         """
         return self._http_request(method="PUT", url_suffix="trust/data", json_data=data, resp_type="response", ok_codes=(200,))
 
-    def get_container_scan_results(self, params: Optional[dict] = None) -> List[dict]:
+    def get_container_scan_results(self, params: Optional[dict] = None, all_results: bool = False) -> List[dict]:
```

Comment:
looks good, add a test where we see that the right numbers are used when paging multiple pages, and another test that it knows to stop when hitting the end